src/DataTest.java

```diff
@@ -0,0 +1,37 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Junit testing for the all the data to be read in the system.
+ * @author Azizul Hasan 101124159
+ * */
+
+class DataTest{
+    
+    @Test
+    void test() throws InterruptedException{
```

Comment:
Could we get a better name for this test function? Also the test cases should test all of the functions in each class individually.

Reply:
I added individual tests to the program